Document the `previous_object`added in Expression Language (#823)
api-platform/docs: core/security.md

 *         "post"={"access_control"="is_granted('ROLE_ADMIN')"}
 *     },
 *     itemOperations={
 *         "get"={"access_control"="is_granted('ROLE_USER') and object.owner == user"},

>  *         "put"={"access_control"="is_granted('ROLE_USER') and previous_object.owner == user"}

Reviewer: Missing trailing comma.

     * @ORM\ManyToOne(targetEntity=User::class)
     */
    public $owner;

    // ...
}
```

This example is only going to allow fetching the book related to the current user. If the user tries to fetch a book which is not
linked to his account, it will not return the resource. In addition, only admins are able to create books which means
that a user could not create a book.

> Additionally, in some cases you need to perform security checks on the original data. For example here, only the actual owner should be allowed to edit its book. In those cases, you can use the `previous object` variable that will contain the object that was read from your datasource.

Reviewer 2: ```suggestion
Additionally, in some cases you need to perform security checks on the original data. For example here, only the actual owner should be allowed to edit its book. In those cases, you can use the `previous_object` variable that will contain the object that was read from your datasource.
```

> Additionally, in some cases you need to perform security checks on the original data. For example here, only the actual owner should be allowed to edit its book. In those cases, you can use the `previous_object` variable that will contain the object that was read from your datasource.

Reviewer: Needs an empty blank line before to start a new paragraph. (Markdown syntax)

```suggestion
Additionally, in some cases you need to perform security checks on the original data. For example here, only the actual owner should be allowed to edit their book. In these cases, you can use the `previous_object` variable which contains the object that was read from the data provider.
```
